.github/workflows/macosx_windows_ci.yaml

```diff
@@ -18,52 +20,44 @@ jobs:
         with:
           fetch-depth: 1
 
-      - name: Install Miniconda
-        shell: bash
-        run: |
-          if [[ "${{ runner.os }}" = "Linux" ]]; then
-            MINICONDA_FILENAME=Miniconda3-latest-Linux-x86_64.sh
-            curl -L -o $MINICONDA_FILENAME "https://repo.continuum.io/miniconda/$MINICONDA_FILENAME"
-            bash ${MINICONDA_FILENAME} -b -f -p $HOME/miniconda3
-          elif [[ "${{ runner.os }}" = "macOS" ]]; then
-            wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O $HOME/miniconda.sh;
-            bash $HOME/miniconda.sh -b -p $HOME/miniconda3
-          elif [[ "${{ runner.os }}" = "Windows" ]]; then
-            MINICONDA_FILENAME=Miniconda3-latest-Windows-x86_64.exe
-            curl -L -o $MINICONDA_FILENAME "https://repo.continuum.io/miniconda/$MINICONDA_FILENAME"
-            ./Miniconda3-latest-Windows-x86_64.exe //InstallationType=JustMe //RegisterPython=0 //S
-          fi
-      - name: Setup paths
-        shell: bash
+      - name: Set ENV NAME
         run: |
           if [[ "${{ runner.os }}" = "Windows" ]]; then
-            echo "::add-path::$HOME/miniconda3"
-            echo "::add-path::$HOME/miniconda3/Scripts"
-            echo "::set-env name=ENV_NAME::pyuvdata_tests_windows"
+            echo "::set-output name=ENV_NAME::pyuvdata_tests_windows"
           else
-            echo "::add-path::$HOME/miniconda3/bin"
-            echo "::add-path::$HOME/miniconda3/envs/test/bin"
-            echo "::set-env name=ENV_NAME::pyuvdata_tests"
+            echo "::set-output name=ENV_NAME::pyuvdata_tests"
+          fi
+        id: env_name
+
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.6.0
+        with:
+          auto-update-conda: "true"
+          miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          environment-file: ci/${{ steps.env_name.outputs.ENV_NAME }}.yml
+          activate-environment: ${{ steps.env_name.outputs.ENV_NAME }}
+
+      - name: Conda Info
+        run: |
+          conda info -a
```

Comment:
I think we also want a `conda list` call, I very often want to see which exact versions of packages were installed.